docs/frontend/in-depth/axios.md

```diff
@@ -0,0 +1,74 @@
+# API requests powered by axios
+
+All API requests are supported by the [nuxt/axios][axios] library,<br>
+which makes it easy for us to use API queries.
+
+### Introduction
+---
+The main configuration for the library is located in the file `nuxt.config.js`, <br>
+there are the most important things you need to run the library on our system. <br>
+
+To configure all necessary settings for API queries, <br>
+you need to overwrite configurations by creating `axios.js` plugin in  **Core** module.<br>
+
+All API queries pass the **authorization token**, <br>
+which is created after the user's login. <br>
+It is placed by default in the header of the `authorization`.<br>
+
+All queries are based on the global `axios` method.<br>
+
+### Structure
+---
+Initially, all necessary headers are added to the axios query.<br>
+Then the two basic **axios** mechanism methods are replaced:<br>
+
+* `onRequest` - this method contains everything that has to be sent with every API query.<br>
+    The query address is prepared here to make it easier to work with the API.<br>
+    Together with the API address the language is also added, which is taken from user data by default.<br>
```

Comment:
Together with the API address the language, which is taken from user data by default, is also added.<br>